libs/designsystem/src/lib/components/data-table/table/table.component.scss

```diff
@@ -40,7 +40,8 @@ th {
 }
 
 td {
-  padding: utils.size('s');
+  height: 44px;
```

Comment:
Would be good to quickly investigate if we expect additional table cell heights in the future. Then we could add this to variables similar to item-heights: 
```
$item-heights: (
  m: 56px,
  s: 44px,
  xs: 32px,
) !default;
```

Or even modify the name of that one if they should be similar!

Reply:
@RasmusKjeldgaard we still haven't received any response whether we expect additional table cel height in the future or not. Should we push for a response or should we add it later, if it becomes necessary? :)

Reply:
Let us extract it to a variable and get on with it! :D